packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -1,30 +1,48 @@
-import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 
 import { setUpZoeForTest } from '@agoric/zoe/tools/setup-zoe.js';
 import { E } from '@endo/far';
 import path from 'path';
+import { makeNodeBundleCache } from '@endo/bundle-source/cache.js';
+import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import type { TestFn } from 'ava';
+import {
+  divideBy,
+  multiplyBy,
+  parseRatio,
+} from '@agoric/zoe/src/contractSupport/ratio.js';
+import { makePromiseKit } from '@endo/promise-kit';
 import { commonSetup } from './supports.js';
 
 const dirname = path.dirname(new URL(import.meta.url).pathname);
 
+const contractName = 'fast-usdc';
 const contractFile = `${dirname}/../src/fast-usdc.contract.js`;
 type StartFn = typeof import('../src/fast-usdc.contract.js').start;
 
+const makeTestContext = async () => {
+  const bundleCache = await makeNodeBundleCache('bundles', {}, s => import(s));
```

Comment:
Unfortunate the test has this burden. I've filed https://github.com/Agoric/agoric-sdk/issues/10427